### PR TITLE
test(meta): testing meta fields

### PR DESCRIPTION
**Pull request by kcvan**
## Description
Why did you write this code?

## How to Test
- [ ] Visit XXX
- [ ] Click Y

Which devices did you test on?
- [ ] Chrome on Mac
- [ ] Chrome on PC
- [ ] Firefox on Mac
- [ ] Firefox on PC
- [ ] Safari iPhone
- [ ] Chrome Android

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
Will this have future work?

## Learnings
Did you learn anything here you want to share with the team?

### CODE REVIEW

File: src/templates/index.development.html [hunk 1]
```diff
@@ -5,15 +5,20 @@
...
+  <title>Beenest | Book Rentals and Homes for your Business</title>
   <meta property="description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
-  <meta property="og:description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
   <meta property="og:title" content="Beenest | Book Rentals and Homes for your Business" />
-  <meta property="og:site_name" content="Beenest" />
+  <meta property="og:type" content="website" />
   <meta property="og:url" content="https://www.beenest.com" />
   <meta property="og:image" content="https://static.beenest.com/images/app/misc/host-cta-image.jpg" />
+  <meta property="og:description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
+  <meta property="og:site_name" content="Beenest" />
```
Comment (tc) [inline src/templates/index.development.html]: can the title "Beenest | Book Rentals and Homes for your Business"

Comment (kcvan) [reply src/templates/index.development.html]: i think the title is already that

Comment (kcvan) [reply src/templates/index.development.html]: ill chage the site_name to see if it propagates
